source/cloud-native-platform/path-to-live/public-dns.html.md.erb

```diff
@@ -16,6 +16,21 @@ The DNS is self-service, managed in the
 follow the [shuttering](shutter.html#pre-requisite) instructions for which records you need to add
 for a new service.
 
+## Pipeline testing
+
+Jenkins pipelines run automated tests against your application in preview/dev by connecting to your application by it's private IP address, bypassing Azure Front Door.
+
+A tool called External DNS is deployed to the preview and dev clusters to automatically add DNS records to private DNS zones to allow Jenkins pipelines to run the automated tests.
+
+If you wish for these tests to be conducted through Azure Front Door, add a CNAME record in the [public DNS repo](https://github.com/hmcts/azure-public-dns/tree/master/environments) that points your application to the front door.
```

Comment:
it's the private DNS repo isn't it that it needs to be in?

Also they need to add it to the frontdoor config as well.

This doesn't belong this early in the page as it's not something core to it.

I'm not sure if this does belong on here either or whether it should belong on a pipeline page or in the cnp-jenkins-library README